ipn/ipnlocal/c2n.go

```diff
@@ -77,6 +77,9 @@ var c2nHandlers = map[methodAndPath]c2nHandler{
 
 	// Linux netfilter.
 	req("POST /netfilter-kind"): handleC2NSetNetfilterKind,
+
+	// VIP services.
+	req("GET /vip-services"): handleC2NVIPServicesGet,
```

Comment:
question (this might be somewhere in a workflow design that I have missed): what is the call of this endpoint expected to look like/when do we expect clients to call it?

Reply:
Ahh, I guess control is meant to call this when a hostinfo with service hash is received